rustfmt.toml

```diff
@@ -1,3 +1,7 @@
 # Imports
 imports_granularity = "Crate"
 group_imports = "One"
+
+# Code
+max_width = 100
+wrap_comments = true
```

Comment:
nit: newline

Reply:
if this looks good, will add the newline after the step "roll back the last commit" in the description